The process 'pwsh' failed with exit code 1

**Describe the bug**

Installation on Windows fails since today with the following error message: The checksum of the downloaded CA certificates is wrong!

**To reproduce**

```yml
on: push

jobs:
    main:
        runs-on: windows-latest
        steps:
            - name: Setup PHP
              uses: shivammathur/setup-php@master
              with:
                  php-version: 7.3
                  extension-csv: dom, filter, gd, hash, intl, json, pcre, pdo, zlib
                  coverage: none
```

**Log**

```
Run shivammathur/setup-php@master
pwsh C:\hostedtoolcache\windows\win32.ps1 -version 7.3 -dir d:\a\_actions\shivammathur\setup-php\master\dist

==> Setup PhpManager 
√ PhpManager Installed 

==> Setup PHP and Composer 
The checksum of the downloaded CA certificates is wrong!
At C:\Users\runneradmin\Documents\PowerShell\Modules\PhpManager\1.19.2.231\private\Get-CACertFromCurl.ps1:35 char:13
+             throw "The checksum of the downloaded CA certificates is  ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : OperationStopped: (The checksum of the\u2026tificates is wrong!:String) [], RuntimeException
+ FullyQualifiedErrorId : The checksum of the downloaded CA certificates is wrong!

##[error]The process 'pwsh' failed with exit code 1
##[error]Node run failed with exit code 1
```

**Additional information**

https://github.com/leofeyer/gha-test/commit/b9c59a05a74bf5bc1632b69eb87891094054d187/checks?check_suite_id=331319247

--- a/src/scripts/linux.sh
+++ b/src/scripts/linux.sh
@@ -20,6 +20,8 @@ existing_version=$(php-config --version | cut -c 1-3)
 version=$1
 status="Switched to PHP$version"
 step_log "Setup PHP and Composer"
+sudo mkdir -p /var/run
+sudo mkdir -p /run/php
 find /etc/apt/sources.list.d -type f -name 'ondrej-ubuntu-php*.list' -exec sudo DEBIAN_FRONTEND=noninteractive apt-fast update -o Dir::Etc::sourcelist="{}" ';' >/dev/null 2>&1
 if [ "$existing_version" != "$1" ]; then
 	if [ ! -e "/usr/bin/php$1" ]; then
@@ -41,7 +43,6 @@ fi
 ini_file=$(php --ini | grep "Loaded Configuration" | sed -e "s|.*:s*||" | sed "s/ //g")
 ext_dir=$(php -i | grep "extension_dir => /usr" | sed -e "s|.*=> s*||")
 sudo chmod 777 "$ini_file"
-sudo mkdir -p /run/php
 add_log "$tick" "PHP" "$status"
 if [ "$2" = "true" ]; then
   if [ "$1" != "7.4" ]; then

--- a/src/scripts/win32.ps1
+++ b/src/scripts/win32.ps1
@@ -53,6 +53,7 @@ catch {
   try {
     Update-PhpCAInfo -Path $php_dir -Source Curl
   } catch {
+    Update-PhpCAInfo -Path $php_dir -Source Curl -SkipChecksumCheck
   }
 }
 if ([Version]$installed.Version -ge '7.4') {

